### PR TITLE
0.10.1 fixes

### PR DESCRIPTION
**Pull request by mingwandroid**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/install-liblief.bat [hunk 1]
```diff
@@ -1,4 +1,2 @@
-pushd build
```
Comment (isuruf) [inline recipe/install-liblief.bat:1]: This needs to be restored as the project is configured inside a folder called build.

Comment (mingwandroid) [reply recipe/install-liblief.bat:1]: I´m undoing the build folder. Missed this one.

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

**Comment by mingwandroid** [comment]
@isuruf, where is the vs2017 package's feedstock? I added setting CC and CXX to the defaults package one now.

**Comment by isuruf** [comment]
@mingwandroid, we are using a copy of defaults's build before the latest due to https://github.com/conda-forge/llvmdev-feedstock/pull/73#issuecomment-533756169
I haven't had time to look into it yet.

**Comment by mingwandroid** [comment]
I'm a bit confused by the Windows failure. It's specific to Python 3.8 and it doesn't matter whether I statically or dynamically link lief.pyd to the lief library (libLIEF.dll when dynamic, my preference).

I suspect something is broken with CMake's FindPython and I am adding some tests to AnacondaRecipe's python 3.8 recipe to try to shake this out. Any insights would be appreciated, pinging @msarahan and @isuruf.

**Pull request closed without merging**